doc/docs/master/deploy-manage/deploy/loki.md

```diff
@@ -0,0 +1,68 @@
+!!! note To deploy and configure a Pachyderm cluster
+to ship logs to Loki, 
+a ***Pachyderm Enterprise License*** is required. 
+
+# Enabling Loki
+
+Enabling Loki logging is done by setting the `LOKI_LOGGING` [environment variable](https://docs.pachyderm.com/latest/deploy-manage/deploy/environment-variables/) to `true` on the pachd container.
```

Comment:
IMO, It would be better to rename this to **Configuring Loki** and cover all required env variables since `LOKI_LOGGING` isn't sufficient. `LOKI_SERVICE_HOST` and `LOKI_SERVICE_PORT` need to be set as well and having all env variables listed in one place will be easier for the reader.

Reply:
Done.  I added the list of environment variables at the beginning of the page for whoever does not wish to read further.